### ISSUE TITLE
[Issue] AGS performance slows down

### ISSUE THREAD

**Issue by yuneguy** [end-4/dots-hyprland#851]
- Linux distro: CachyOS

---
## The issue
At the end of the day, the right and left panels open with a noticeable delay. What I do for this: I watch a little game playing, programming in vs code, hardware and I are big enough,

processor: 8-core Intel Xeon E5-2680 v3 (-MT MCP-) speed/min/max: 1200/1200/3300 MHz
Core: 6.11.0-4-cachyos-barex86_64, Memory capacity: 2.87/15.46 GB (18.6%)
Storage capacity: 238.47 gigabytes (68.3% used) shell: fish

ananicyc-cpp is worth, but I don't think it has any effect on this.

helps to restart the config

**Comment by NotAF0e** [comment]
I have the same (or a similar) issue. It seems to be weird cpu load. The app launcher loads slower, apps take longer to open and everything feels sluggish. For me the fix is to restart the whole system

**Comment by s0mesh1t** [comment]
Yes, it's because of the music module, I completely cut out the wifu module

**Comment by BoozyAF** [comment]
Having this same issue. Restarting Ags fixes it, but it is annoying. Since images were mentioned, how could I disable the media player images from even being used?

**Comment by end-4** [comment]
Stale as the shell has been rewritten to use Quickshell instead of AGS
